scripts/fastlane/Fastfile

```diff
@@ -0,0 +1,128 @@
+
+    git_add(path: [gradle_properties_path, change_log_path])
+    git_commit(path: [gradle_properties_path, change_log_path], message: "chore(release): #{product_name} - #{next_version}")
+    push_to_git_remote(force: true)
+
+
+    current_branch = sh('git', 'branch', '--show-current')
+    repo_name_parts = options[:repo].split('/')
+    create_pull_request(base: 'main', 
+                        title: "chore(release): #{product_name} - #{next_version}", 
+                        body:pr_body, 
+                        repo: options[:repo], 
+                        api_token: ENV["RELEASE_MANAGER_TOKEN"])
+  end
+
+  desc "Update changelog"
+  private_lane :update_change_log do |options|
+    # We won't display the title generated by conventional_changelog because it will have the 
+    # incorrect version.
+    changelog_body = notes = conventional_changelog(display_title: false)
+
+    open(change_log_path, 'a') { |f|
+      f.puts "# Release #{options[:next_version]}\n\n"
+      f.puts "#{changelog_body}\n\n"
+    }
+  end
+
+  desc "Increment versions"
+  private_lane :update_gradle_properties do |options|
+    version = options[:version].to_s
+    segments = version.match(version_regex).captures # version.split('.')
+
+    UI.message("Updating versionName in gradle.properties")
+
+    set_key_value(file: gradle_properties_path, key: 'VERSION_NAME', value: version)
+
+  end
+
+
+  desc "Create the release commit."
+  private_lane :release_commit do |options|
+    next_version = options[:version]
+    commit_message = "chore: release #{next_version} [skip ci]"
+    sh('git', 'commit', '-am', commit_message)
+  end
+
+  desc "Publish release to maven using attributes from environment variables."
+  private_lane :publish_to_maven do |options|
+    gradle(
+      project_dir: gradle_project_root,
+      task: "uploadArchives",
+      print_command: false,
+      flags: '--stacktrace',
+      properties: {
+        "SONATYPE_NEXUS_USERNAME": ConfigValueReader.from_environment_variable('SONATYPE_NEXUS_USERNAME'),
+        "SONATYPE_NEXUS_PASSWORD": ConfigValueReader.from_environment_variable('SONATYPE_NEXUS_PASSWORD'),
+        "signing.inMemoryKey": ConfigValueReader.from_environment_variable('signing.inMemoryKey'),
+        "signing.password": ConfigValueReader.from_environment_variable('signing.password'),
+        "signing.keyId": ConfigValueReader.from_environment_variable('signing.keyId'),
+        "signing.secretKeyRingFile": ConfigValueReader.from_environment_variable('signing.secretKeyRingFile')
+      }
+    )
+  end
+end
+
+module ConfigValueReader
+  def self.from_environment_variable(param_name)
+    ENV[param_name.gsub('.','_')]
+  end
+
+  def self.from_ssm(param_name)
+    result = `aws ssm get-parameter --name #{param_name} --with-decryption --region us-east-1 --query Parameter.Value --output text`
+    if $?.exitstatus != 0
+      raise "Failed to get value for parameter #{param_name}"
+    end
+    result
+  end
+end
```

Comment:
Unused. 
```suggestion
```

Reply:
Correct. Since we don't need fastlane to execute the uploadArchives task, all this stuff can go.